README.md

```diff
@@ -14,6 +14,8 @@ This repository contains several code and data components:
 
 + `pubchem-map.ipynb` -- DrugBank compounds were mapped to [PubChem](https://pubchem.ncbi.nlm.nih.gov/search/) based on exact InChi string matches. The mapping is available as a [tsv file](data/pubchem-mapping.tsv).
 
++ `parse-halflife.ipynb` -- extracts half-life and other structural information from the Drugbank xml download into a [tsv file](data/drugbank_halflife.tsv) where each row represents a drug. The half-life information was listed as free text in Drugbank. We manually extract the numeric value from free text into a [xlsx file](data/drugbank_halflife_curated.xlsx).
```

Comment:
Should we add a little description here on the methodology of the manual curation of `half_life`. For example, what method did you use when there was ambiguity or a range?